src/js/shield_defs.js

```diff
@@ -590,16 +591,6 @@ export function loadShields() {
         },
       })
   );
-  shields["MX:JAL"] = {
```

Comment:
In that case, we should keep #411 open rather than declaring it fixed, to allow for more evidence to be gathered.